knative-operator/Gopkg.toml

```diff
@@ -1,8 +1,3 @@
-# Force dep to vendor the code generators, which aren't imported just used at dev time.
```

Comment:
What was the original motivation to force this dependency for development and why is this not necessary anymore ?

Reply:
No clue, I don't think it was ever needed. There's no code generation here.